### DISCUSSION TITLE
Is it possible to get a route's middlewares?

### DISCUSSION THREAD

**Discussion by alannaidon** [adonisjs/core#1887] [Help]
I already know how to create and register a middleware, but now I need something more.  
We can generate an AuthToken:jwt to access the system as a different user.
I need to ensure that some of my routes are NOT accessible with this kind of token.  

I already know how to get the header information (Token,  authorization and AccessKey). But I need some kind of FLAG which will tell me if that route can/can't be accessed with this kind of token.  

I created middleware 'checkVraw', is it possible to check if the route has a different middleware?  
Or maybe pass a prop for global middleware? 

AdonisJS version `4.1.0` 

In `Express` we can pass a value between `middlewares`, tried but failed. Can we do that with Adonis?  Something like: 
```
request.locals.variable = myVariable
```  

thanks

**Comment by thetutlage** [comment]
Which version of the framework are you using?

**Reply by alannaidon** [reply]
> Which version of the framework are you using?

Sorry. It's `4.1.0`
